### DISCUSSION TITLE
Configuring Logs: Log file is empty

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#86] [Q&A] [answered]
### Current Behavior

log4j-sbb-20240710-0757-36.log is created in the folder "C:\Polarion\data\logs\main" but file is always empty. it is not capturing SBB pdf export error in separately.

all the required logging configs are made. Please find log4j2.xml (*.zipi) file attached for my local config.
[log4j2.zip](https://github.com/user-attachments/files/16155012/log4j2.zip)

### Expected Behavior

as per documentation: It is also possible to write all messages of SBB extensions info separate log file which can be useful to report a problem

### Steps To Reproduce

try to check log4j-sbb-DATE_TIME.log for the errors if it is capturing pdf export erros.

### Environment - OS

windows

### Polarion version

2404

### Extension Version

https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/releases/tag/v5.5.0

**Comment by ukrishne** [comment]
eg:issue mentioned in the https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/issues/77 are not stored at log4j-sbb${fileNameSuffix}

**Comment by grigoriev** [comment, marked as answer]
Hi @ukrishne,

looks like you have wrong configuration of log4j2.xml:
<img width="830" alt="image" src="https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/4573001/a924990b-25a2-471c-9652-a84e072f9be3">

Could you please remove the line 170, restart Polarion and check the log file again.

And it's not a bug of PDF-exporter at all. It's related for Polarion configuration.
